src/unix/utilsunx.cpp

```diff
@@ -202,6 +204,33 @@ void wxMilliSleep(unsigned long milliseconds)
     wxMicroSleep(milliseconds*1000);
 }
 
+// ----------------------------------------------------------------------------
+// security
+// ----------------------------------------------------------------------------
+
+void wxSecureZeroMemory(void* v, size_t n)
+{
+#if (defined(__GLIBC__) && \
+        (__GLIBC__ > 2 || (__GLIBC__ == 2 && __GLIBC_MINOR__ >= 25))) || \
+    (defined(__FreeBSD__) && __FreeBSD__ >= 11)
```

Comment:
Wouldn't it be better to check for this function in configure instead?

And if we keep the glibc check here we might want to add `wxCHECK_GLIB_VERSION()` similar to the existing `wxCHECK_GCC_VERSION()` to make it a bit more palatable to write it.

Reply:
No doubt configure/CMake is the more correct way to check availability, but I am hoping this `#if` expression is a reliable enough way to detect it, as adding configure/CMake checks isn't quite trivial either. (Surely takes more than 3 lines.)